tests/v1/entrypoints/llm/test_struct_output_generate.py

```diff
@@ -29,19 +30,20 @@
     "num_speculative_tokens": 5,
 }
 
-PARAMS_MODELS_BACKENDS_TOKENIZER_MODE = [
-    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "auto", None),
-    ("mistralai/Ministral-8B-Instruct-2410", "guidance", "auto", None),
-    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "mistral", None),
-    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar", "auto", None),
+# yapf: disable
+PARAMS_MODELS_BACKENDS_TOKENIZER_MODE_REASONING_PARSER_SPEC_CONFIG = [
+    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "auto", None, None),
+    ("mistralai/Ministral-8B-Instruct-2410", "guidance", "auto", None, None),
+    ("mistralai/Ministral-8B-Instruct-2410", "xgrammar", "mistral", None, None),
+    ("Qwen/Qwen2.5-1.5B-Instruct", "xgrammar", "auto", None, None),
+    ("deepseek-ai/DeepSeek-R1-Distill-Qwen-1.5B", "xgrammar", "auto", "deepseek_r1", None),
+    ("Qwen/Qwen3-0.6B", "xgrammar", "auto", "qwen3", None, NGRAM_SPEC_CONFIG),
```

Comment:
Is this new test case related to thinking compatibility? It doesn't look like it?